app/editor/src/features/admin/reports/constants/defaultReport.ts

```diff
@@ -23,6 +23,7 @@ export const defaultReport: IReportModel = {
       showSentiment: false,
     },
     content: {
+      clearFolders: false,
```

Comment:
This was throwing an error for me, just want to confirm this is ok,

Reply:
yes thanks.  I was surprised it wasn't throwing an error for me.  I have it fixed in my current work.